docs/network_disruption/prio.md

```diff
@@ -152,6 +152,43 @@ We first filter for all packets related to `gateway IP` and `node IP` and send t
 
 Finally, we apply a filter to enqueue all packets to class `1:4` whenever the `destination IP` is encompassed by the `hosts` field (see [this documentation](../../docs/network_disruption/hosts-and-services.md) for more details). In this case, a filter is applied for `10.0.1.254/32` and another for `10.0.1.255/32`. If no hosts were specified, a single filter is applied for `0.0.0.0/0` and no traffic is ends up in class `2:1`.
 
+### Network Disruption implementation for pod level with eBPF filters
+
+Tc can use eBPF filters in order to intercept packets at the application layer and allow the access to the payload of packets.
+Network disruption used eBPF filters to apply disruption by filtering with `method` and/or `path`:
+
+Now, let us take the following pod level disruption spec with new filters:
+```
+spec:
+  level: pod
+  selector:
+    app: demo
+  count: 1
+  network:
+    hosts:
+      - 10.0.1.254/31
+    port: 80
+    protocol: tcp
+    method: get <-- eBPF filter
+    path: /test <-- eBPF filter
```

Comment:
This is a fairly sticky (as in hard to change) decision as to where to put these options. I think having only one each per disruption might be too inflexible? It's completely plausible to imagine someone wanting to have a small list of these, no?

`network` has always been L4 disruptions. If we're adding L7, do we not want them to be a different kind entirely?

Reply:
I agree with you, it could be nice to be able to define a list of **path** and **method** to apply the disruption, but I just do something simple for first PR to simplify the review. Currently, it is not possible at all to filter on `method` and `path`. This is the same point for the disk failure injection based on eBPF.

What do you mean by a `different kind entirely`? Do you have an example?

Reply:
`Kind` as in an entirely new top level disruption type, alongside network, e.g.
```
spec:
  network:
    ...
  http:
    ...
```
    it's just an idea though

Reply:
Yes, given the implementation here, I do think something like https://github.com/DataDog/chaos-controller/pull/741#discussion_r1266427453 is preferable

Reply:
I have updated the PR to defined filters like https://github.com/DataDog/chaos-controller/pull/741#discussion_r1266427453 👍